src/Razor/test/Microsoft.VisualStudio.Editor.Razor.Test/TelemetryReporterTests.cs

```diff
@@ -239,6 +240,58 @@ public void Block_FourArguments()
             });
     }
 
+    [Fact]
+    public void HandleRIEWithInnerException()
+    {
+        var reporter = new TestTelemetryReporter();
+
+        var ae = new ApplicationException("expectedText");
+        var rie = new RemoteInvocationException("a", 0, ae);
+
+
+        var p3Value = Guid.NewGuid();
+        reporter.ReportFault(rie, rie.Message);
+
+        Assert.Collection(reporter.Events,
+            e1 =>
+            {
+                Assert.Equal(TelemetrySeverity.High, e1.Severity);
+                Assert.Equal("dotnet/razor/fault", e1.Name);
+                if (e1 is not FaultEvent faultEvent1)
+                {
+                    // faultEvent doesn't expose any interesting properties,
+                    // like the ExceptionObject, or the resulting Description,
+                    // or really anything we would explicitly want to verify against.
+                    Assert.Fail("It should be a fault event");
+                }
+            });
+    }
+
+    [Fact]
+    public void HandleRIEWithNoInnerException()
+    {
+        var reporter = new TestTelemetryReporter();
+
+        var rie = new RemoteInvocationException("a", 0, errorData:null);
+
+        var p3Value = Guid.NewGuid();
```

Comment:
Nit: Same as above, doesn't seem to get used?

Reply:
Fixed.